### PR TITLE
feat(select): homogenization - INNO-756

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

Homogenization of component.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] `package.json` is up-to-date and `@ec-europa/ecl-base` is part of the dependencies
- [ ] I have checked the dependencies
- [ ] I have given the fractal status “ready” to my component
- [ ] I have declared `@define mycomponent` in the SCSS file
- [ ] I have specified `margin: 0;` on the CSS component
- [ ] I have provided tests
- [ ] I follow the naming guidelines
- [ ] the component supports composition
- [ ] there are no hardcoded strings (all content come from the context)
- [ ] I have filled the README.md file (at least a few lines)
- [ ] My component is listed in the root README
- [ ] My PR has the right label(s)

### CODE REVIEW

File: framework/components/ecl-forms/ecl-forms-selects/ecl-forms-selects.twig [hunk 1]
```diff
@@ -12,8 +21,8 @@
   {% set _css_class = _css_class ~ ' ecl-select--has-error' %}
 {% endif %}
 
-{% if extra_class is defined %}
-  {% set _css_class = _css_class ~ ' ' ~ extra_class %}
+{% if extra_classes is defined %}
```
Comment (yhuard) [inline framework/components/ecl-forms/ecl-forms-selects/ecl-forms-selects.twig]: Sometimes we use `extra_class`, sometimes `extra_classes`. Can we agree to use `extra_classes` everywhere?

**Review by yhuard — CHANGES_REQUESTED** [review]
I would also remove the content of the `options` block. A list of options could be provided instead

File: framework/components/ecl-forms/ecl-forms-selects/ecl-forms-selects.twig (hunk 1 above)
Comment (degliwe) [inline framework/components/ecl-forms/ecl-forms-selects/ecl-forms-selects.twig]: Let's pick one (I can live with any option) and stick to it.

Comment (kalinchernev) [inline framework/components/ecl-forms/ecl-forms-selects/ecl-forms-selects.twig]: `extra_classes` matches better I think.

**Comment by emeryro** [comment]
Options set as variables.
Ready for review

File: framework/components/ecl-forms/ecl-forms-selects/ecl-forms-selects.config.js [hunk 2]
```diff
@@ -26,6 +35,7 @@ module.exports = {
       context: {
         id: 'example-selectid-2',
         is_disabled: true,
+        options: optionsList,
```
Comment (yhuard) [inline framework/components/ecl-forms/ecl-forms-selects/ecl-forms-selects.config.js]: Small tip: you can define a common context to all variants (with `context`). Check http://fractal.build/guide/core-concepts/configuration-files#configuration-inheritance

**Review by yhuard — APPROVED** [review]

**Pull request merged**